Formula/bit.rb

```diff
@@ -2,15 +2,8 @@
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "http://assets.bitsrc.io/release/0.1.0/bit_0.1.0_brew.tar.gz"
-  sha256 "c33c00a8334da4659b263472d013eaafbc5f0c16249e755f7624ac396d673270"
-
-  bottle do
```

Comment:
You don't need to remove this; `test-bot` will automatically rewrite these lines.